.eslintrc.js

```diff
@@ -231,6 +238,35 @@ module.exports = {
                 'no-console': 'off',
             },
         },
+        {
+            // we are using explicit blacklist because this will enforce new rules in newly created packages
+            files: [
+                'packages/connect/**/*',
+                'packages/connect-common/**/*',
+                'packages/connect-explorer/**/*',
+                'packages/connect-web/**/*',
+                'packages/connect-popup/**/*',
+                'packages/connect-iframe/**/*',
+                'packages/connect-examples/**/*',
+                'packages/connect-plugin-ethereum/**/*',
+                'packages/connect-plugin-stellar/**/*',
+                'packages/integration-tests/**/*',
+                'packages/request-manager/**/*',
+                'packages/suite-desktop-api/**/*',
+                'packages/suite-storage/**/*',
+                'packages/suite-web-landing/**/*',
```

Comment:
does not exist anymore

Reply:
4641dcfcb78cd5701a1045984c285afc67963f8b